src/App.tsx

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    lastKey: '',
+    listener: (event: KeyboardEvent) => {
+      this.setState({ lastKey: event.key });
+    },
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.state.listener);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.state.listener);
+  }
+
+  render() {
+    return (
+      <div
+        className="App"
+      >
```

Comment:
no need for multi-line here
```suggestion
      <div className="App">
```